source/indexes/single-field-index.txt

```diff
@@ -0,0 +1,59 @@
+.. _pymongo-single-field-index:
+
+====================
+Single Field Indexes
```

Comment:
I'm about 80% sure this should be "Single-Field Indexes." If you make the change, applies to all occurrences